src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.css

```diff
@@ -0,0 +1,308 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.ads-initial-setup-wizard {
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	width: 719px;
+	z-index: 9;
+	background: transparent;
+}
+
+.ads-initial-setup-show {
+	display: block;
+}
+
+.ads-initial-setup-hide {
+	display: none;
+}
+
+.ads-initial-setup-wizard:focus {
+	outline: 0 none !important;
+}
+
+.ads-initial-setup-wizard-text-container {
+	padding: 0 24px 22px 24px;
+}
+
+.ads-initial-setup-wizard h1 {
+	font-size: 18px;
+	font-weight: 700;
+}
+
+.ads-initial-setup-wizard h2 {
+	font-size: 16px;
+	font-weight: 700;
+	line-height: 22px;
+	margin: 24px 0 12px;
+}
+
+.monaco-workbench > .ads-initial-setup-wizard {
```

Comment:
Why this selector here? Seems odd to have only a few be so specific (and should be unnecessary since the class name is pretty unique)